### PR TITLE
[bitnami/external-dns] Add support for Akamai provider

### LINKED ISSUE #12155: external-dns chart has no support for akamai
### Name and Version

bitnami/external-dns 0.12.2

### What is the problem this feature will solve?

it will add akamai as provider as supported by external-dns: https://kubernetes-sigs.github.io/external-dns/v0.12.2/tutorials/akamai-edgedns/

### What is the feature you are proposing to solve the problem?

add config?

### What alternatives have you considered?

_No response_

### PR DESCRIPTION
**Pull request by joshjohnson**
### Description of the change

Adds support for the Akamai provider supported by external-dns: https://kubernetes-sigs.github.io/external-dns/v0.12.2/tutorials/akamai-edgedns/

### Benefits

Allows Akamai EdgeDNS users to integrate through external-dns

### Applicable issues

  - fixes #12155

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/external-dns/Chart.yaml [hunk 1]
```diff
@@ -24,4 +24,4 @@ sources:
   - https://github.com/kubernetes-sigs/external-dns
   - https://github.com/bitnami/containers/tree/main/bitnami/external-dns
   - https://github.com/kubernetes-sigs/external-dns
-version: 6.8.2
+version: 6.9.2
```
Comment (migruiz4) [inline bitnami/external-dns/Chart.yaml:27]: Please change version to `6.9.0`

File: bitnami/external-dns/templates/deployment.yaml [hunk 2]
```diff
@@ -290,6 +291,27 @@ spec:
...
+              value: {{ .Values.akamai.clientToken }}
+            {{- end }}
+            {{- if or (.Values.akamai.clientSecret) (.Values.akamai.secretName) }}
+            - name: EXTERNAL_DNS_AKAMAI_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "external-dns.secretName" . }}
+                  key: akamai_client_secret
+            {{- end }}
+            {{- end }}
```
Comment (migruiz4) [inline bitnami/external-dns/templates/deployment.yaml:314]: I think it is missing here an opening `{{- if eq .Values.provider "akamai" }}`, the number of if/end does not match.

File: bitnami/external-dns/templates/deployment.yaml [hunk 3]
```diff
@@ -161,6 +161,7 @@ spec:
             {{- range .Values.sources }}
             - --source={{ . }}
             {{- end }}
+            {{- if eq .Values.provider "akamai" }}
```
Comment (migruiz4) [inline bitnami/external-dns/templates/deployment.yaml:164]: No closing end.
Missing akamai arguments.

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Hi @joshjohnson,

Thank you very much for your contribution! Could you please take a look at my comments?

File: bitnami/external-dns/Chart.yaml (hunk 1 above)
Comment (joshjohnson) [reply bitnami/external-dns/Chart.yaml]: Fixed in latest update

File: bitnami/external-dns/templates/deployment.yaml (hunk 3 above)
Comment (joshjohnson) [reply bitnami/external-dns/templates/deployment.yaml]: Looks like I had a merge error and the beginning "if" got separated from the end.  Fixed now.

File: bitnami/external-dns/templates/deployment.yaml (hunk 2 above)
Comment (joshjohnson) [reply bitnami/external-dns/templates/deployment.yaml]: Looks like I had a merge error and the beginning "if" got separated from the end.  Fixed now.

**Comment by joshjohnson** [comment]
Thanks, @migruiz4.  I fixed the version number and the if statement.

**Review by migruiz4 — APPROVED** [review]
@joshjohnson looks good to me, thank you for your contribution!

**Pull request merged**